src/connectdlg.cpp

```diff
@@ -41,51 +41,63 @@ CConnectDlg::CConnectDlg ( CClientSettings* pNSetP, const bool bNewShowCompleteR
     setupUi ( this );
 
     // Add help text to controls -----------------------------------------------
-    // server list
+    // directory
+    QString strDirectoryWT = "<b>" + tr ( "Directory" ) + ":</b> " +
+                             tr ( "Selects the server list to be shown. "
+                                  "You can add custom directories in Advanced Settings." );
+    QString strDirectoryAN = tr ( "Directory combo box" );
+
+    lblList->setWhatsThis ( strDirectoryWT );
+    lblList->setAccessibleName ( strDirectoryAN );
+    cbxDirectoryServer->setWhatsThis ( strDirectoryWT );
+    cbxDirectoryServer->setAccessibleName ( strDirectoryAN );
+
+    // filter
+    QString strFilterWT = "<b>" + tr ( "Filter" ) + ":</b> " +
+                          tr ( "The server list is filtered by the given text. Note that the filter is case insensitive. "
+                               "A single # character will filter for those servers with at least one client connected." );
+    QString strFilterAN = tr ( "Filter edit box" );
+    lblFilter->setWhatsThis ( strFilterWT );
+    edtFilter->setWhatsThis ( strFilterWT );
+
+    lblFilter->setAccessibleName ( strFilterAN );
+    edtFilter->setAccessibleName ( strFilterAN );
+
+    // show all mucisians
+    chbExpandAll->setWhatsThis ( "<b>" + tr ( "Show All Musicians" ) + ":</b> " +
+                                 tr ( "If you check this check box, the musicians of all servers are shown. If you "
+                                      "uncheck the check box, all list view items are collapsed." ) );
+
+    chbExpandAll->setAccessibleName ( tr ( "Show all musicians check box" ) );
+
+    // server list view
     lvwServers->setWhatsThis ( "<b>" + tr ( "Server List" ) + ":</b> " +
-                               tr ( "The Connection Setup window shows a list of available servers. "
-                                    "Server operators can optionally list their servers by music genre. "
-                                    "Use the List dropdown to select a genre, click on the server you want "
-                                    "to join and press the Connect button to connect to it. Alternatively, "
-                                    "double click on on the server name. Permanent servers (those that have "
-                                    "been listed for longer than 48 hours) are shown in bold." ) );
+                               tr ( "The Connection Setup window lists the available servers registered with "
+                                    "the selected directory. Use the Directory dropdown change directory, "
```

Comment:
```suggestion
                                    "the selected directory. Use the Directory dropdown to change directory, "
```

Reply:
... is the "the" really necessary..?  (Like, I pushed the change without it... 😃 )